lua/cfc_random_spawn/module/sv_player_spawning.lua

```diff
@@ -66,13 +125,16 @@ function CFCRandomSpawn.handlePlayerSpawn( ply )
     if not ( ply and IsValid( ply ) ) then return end
     if IsValid( ply.LinkedSpawnPoint ) then return end
 
+    ply.isCFCRespawning = true
```

Comment:
This variable is pretty generic might wanna use something like `ply.CFCRandomSpawnRespawning = true` remember that all addons share the same player table so any cfc addon that'd do something with respawning *could* conflict with this. Very unlikely to happen though.